src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/FlowRulesSpec.groovy

```diff
@@ -742,9 +742,10 @@ class FlowRulesSpec extends HealthCheckSpecification {
             [examVlanFlow.forward, examVlanFlow.reverse].eachParallel { direction ->
                 def resources = traffExam.startExam(direction)
                 direction.setResources(resources)
-                assert traffExam.waitExam(direction).hasTraffic()
+                assert traffExam.waitExam(direction)
             }
         }
+        sleep(4000) //packetCount can be written with some delay
```

Comment:
you can do 'wait' on below operation instead of bare sleeping

Reply:
We can't. We don't know exact number of packetCount which should we wait.

Reply:
no need to change any verifications. Just wrap the below 'then' inside wait.